<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd"><html><head>  <title>Dana Pe'er Lab </title>  </head>  <body>   <img src="../pics/siglogo.jpg" width="980" height="250" border="0"><table width="700" align="center">	 <tbody>		<TABLE><TR valign="top">  <TD><TABLE  border="0"  cellpadding="4" cellspacing="4"><TR><TD bgcolor="#6699cc" align="center">  <a href="../index.html"  style="text-decoration:none"><b><font color="white" face="Arial" size="3">Home</b></a></td></tr><TR><TD bgcolor="#6699cc" align="center">  <a href="../pub/index.html"  style="text-decoration:none"><b><font color="white" face="Arial" size="3">Publications</b></a></td></tr><TR><TD bgcolor="#6699cc" align="center">  <a href="index.html" style="text-decoration:none"><b><font color="white" face="Arial" size="3">Research</b></a></td></tr><TR><TD bgcolor="#6699cc" align="center">  <a href="../people.html" style="text-decoration:none">  <b><font color="white" face="Arial" size="3">People</b></a></td></tr><TR><TD bgcolor="#6699cc" align="center">  <a href="../collaborations.html" style="text-decoration:none">  <b><font color="white" face="Arial" size="3">Collaborations</b></a></tr></td><TR><TD bgcolor="#6699cc" align="center">  <a href="../openings.html" style="text-decoration:none">  <b><font color="white" face="Arial" size="3">Openings</b></a></tr></td><TR><TD bgcolor="#6699cc" align="center">  <a href="../contact.html" style="text-decoration:none">  <b><font color="white" face="Arial" size="3">Contact</b></a></tr></td></TABLE><td width=30></td><TD width=800><img src="../pics/flow.jpg" width="700" height="180" border="0"> <br>Although pathways are often conceptualized as distinct entities responding to specific triggers, it is now appreciated that extensive inter-pathway crosstalk, feedback and other properties (e.g. developmental or metabolic state) of networks reflect underlying complexities that cannot be explained by consideration of individual pathwaysin isolation. Thus, gaining a global understanding of the network and its function requires integration of data from diverse high-throughput genomic and proteomic technologies into one holistic framework. <br>Understanding "cellular computation" requires knowledge of the network structure and the influences among its components.  In collaboration with <a href="http://www.stanford.edu/group/nolan/"> Garry Nolan </a>at Stanford University and <a href="http://web.mit.edu/dallab/"> Doug Lauffenburger </a> at MIT we demonstrated the feasibility of<a href="science2005.pdf"> directly deriving signaling networks </a> from raw flow cytometry data using <a href="../pub/stke2005.pdf"> Bayesian networks </a>.Our approach relied on using<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=15047199&query_hl=5&itool=pubmed_docsum">intracellular multicolor flow cytometry </a> to quantitatively and simultaneously measure the abundance of phosphorylated proteins and phospholipid components in thousands of individual primary human immune cells.Perturbing these cells with molecular interventions drove   the inference of influence and causality between them. Our method automatically discovered de novo, most traditionally established influences between the measured signaling components as well as discovering novel inter-pathway crosstalk: a causal connection of Erk1 on Akt that we confirmed experimentally.   The network was correctly and rapidly reverse-engineered with no a priori knowledge of pathway connectivity. The application of Bayesian networks to single cell flow cytometry has distinct advantages, including an ability to measure events in primary cells after in vivo interventions (measuring context specific signaling biology in tissues), inference of causal influences and the ability to detect indirect as well as direct connections.  Our methods success is due to two key factors:<ul><li> Statistical correlations and dependencies between molecular components often represent molecular interactions and causal influences between them.  This principleholds both for proteins, as well as <a href="../pub/ismb2001.pdf">gene expression </a>.<li> Flow cytometry can measure thousands of individual cells, each an independent experimental sample, providing statistical power to gain information of significant accuracy.</ul>Reconstruction of network models from physiologically relevant primary single cells might be applied to understanding native-state tissue signaling biology, complex drug actions, as well as dysfunctional signaling in diseased cells. <p>We continue to use flow cytometry data as well as data from other proteomic and genomic technologies to address the central question:How does the cell integrate multiple exogenous and endogenous signals from different pathways to compute an appropriate response?   We address this question boththrough experimental collaborations and using publicly available data.<ul><li> We are pushing the computational envelope, enhancing Bayesian network techniques and better adapting them to model molecular signaling. We plan to derive more quantitative models that explicitly depict the logic, activation threshold and the magnitude of each influence.   Moreover, we aim to integrate both proteomic and gene expression data to incorporate both the "calculation" and its resulting output in a single framework. <li> One of the major puzzles in signaling is that the same set of core signaling cascades are activated in response to diverse stimuli and regulate a diverse set of processes in a stimuli and cell-specific manner.  Our ability to measure context specific events in primary cells, after in vivo stimuli, makes it possible to underline these differences.  We continue to collaborate with the <a href="http://www.stanford.edu/group/nolan/">Nolan lab </a>in generating a large scale signaling landscape map that compares signalingin different cell types and under different stimuli. </ul>      </td></table></body></html>