<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd"><html><head>  <title>Dana Pe'er Lab </title>  </head>  <body>   <img src="../pics/cancerlogo.jpg" width="980" height="250" border="0"><table width="700" align="center">	 <tbody>  		<TABLE><TR valign="top">  <TD><TABLE  border="0"  cellpadding="4" cellspacing="4"><TR><TD bgcolor="#6699cc" align="center">  <a href="../index.html"  style="text-decoration:none"><b><font color="white" face="Arial" size="3">Home</b></a></td></tr><TR><TD bgcolor="#6699cc" align="center">  <a href="../pub/index.html"  style="text-decoration:none"><b><font color="white" face="Arial" size="3">Publications</b></a></td></tr><TR><TD bgcolor="#6699cc" align="center">  <a href="index.html" style="text-decoration:none"><b><font color="white" face="Arial" size="3">Research</b></a></td></tr><TR><TD bgcolor="#6699cc" align="center">  <a href="../people.html" style="text-decoration:none">  <b><font color="white" face="Arial" size="3">People</b></a></td></tr><TR><TD bgcolor="#6699cc" align="center">  <a href="../collaborations.html" style="text-decoration:none">  <b><font color="white" face="Arial" size="3">Collaborations</b></a></tr></td><TR><TD bgcolor="#6699cc" align="center">  <a href="../openings.html" style="text-decoration:none">  <b><font color="white" face="Arial" size="3">Openings</b></a></tr></td><TR><TD bgcolor="#6699cc" align="center">  <a href="../contact.html" style="text-decoration:none">  <b><font color="white" face="Arial" size="3">Contact</b></a></tr></td></TABLE>
<td width=30></td><TD width=800>


<img src="../pics/CellCover.png" width="200" height="250" border="0"> 
<img src="../pics/design.png" width="500" height="250" border="0"> 
 <br>
Cancer is an individual disease unique in how it develops and behaves in every patient.  Genetic alterations give rise to <i> cancer networks </i> 
regulation going awry in myriad ways, arming the cancer with ability to abnormally grow, metastasize and evade drugs.  With the advent of <a href="http://cancergenome.nih.gov/index.asp">
The Cancer Genome Atlas </a> and other large-scale cancer genomics projects, genetic and genomic data in tumor samples is accumulating at astounding rates, creating a tsunami of data to mine towards a better understanding of this disease and potential therapeutic paths. Computation has emerged as a bottleneck on the path to personalized therapy, key challenges being: <ul>
<li> Can we identify the causal mutations driving tumor progression and drug resistance? 
<li> Can we assemble these faulty components into functional <i> cancer networks </i> and understand the new cellular dynamics?
<li> Can we derive models that can predict the best drug regime for each individual? 
</ul>

We develop machine learning algorithms to infer <i> cancer networks </i> that elucidate the pathways and regulatory mechanisms by which cancer goes awry and suggest new pharmacological targets for this deadly malignancy in a patient specific manner. By developing models that can predict how individual tumors will respond to certain drugs and drug combinations, our goal is to develop ways to determine the best drug regime for each patient. Success in this challenge will carry cancer genomics from the diagnostic to the therapeutic realm.  <br> <br>

<table>
<TD width=260>
<img src="../pics/CONEXIC.jpeg" width="210" height="280" border="0">  </td>
<td>
<b><u> Driving Mutations </u></b> Systematic characterization of cancer genomes has revealed a staggering complexity of aberrations among individuals, therefore the first step is to separate the wheat from the chaff and define what are the key alterations functionally driving cancer, and to then begin to identify their mechanisms of action and therapeutic implications.  We develop novel computational approaches to do just that. The approaches we develop can be viewed as an accurate in silico screens that concurrently works across multiple genetic backgrounds and cancer phenotypes, accelerating discovery to rapidly extract the maximal value from modern genomic studies.  <br>
Our first algorithm, <a href="http://www.sciencedirect.com/science/article/pii/S0092867410012936"> CONEXIC </a>, integrates DNA copy number and gene expression to pinpoint specific drivers within large aberrant regions and provides insight into their physiological roles. We applied CONEXIC to melanoma and demonstrated that we could accurately identify the tumor dependencies and predict which pathways and biological processes these altered. <br>
We continue to push the envelop, developing more sophisticated machine learning algorithms that integrate additional data types such as sequence mutations, epigenetics and functional shRNA screens to identify oncogenes and tumor suppressors in a tumor specific manner. Our current projects focus on the genetic determinants of drug resistance and modeling interactions between multiple genetic lesions. 
</td>
</table><table>
<TD width=260>
<img src="../pics/meki.jpg" width="250" height="250" border="0">  </td>
<td>  
<b><u> Cancer networks and personalized therapy </u></b>
Success of targeted therapies relies on our ability to match these with the right patients. As it becomes more evident that effective treatments will require combinatorial regimes of drugs, rational therapy becomes an increasingly complex challenge. Rather than <i> black box </i> predictors of drug response, we take a <i> network centric </i> approach, building models that can identify genetic determinants of drug response, how and why these influence drug response and pinpoint additional targets that are likely to enhance sensitivity to drug.  To construct such <a href = "http://www.sciencedirect.com/science/article/pii/S0092867411002376"> cancer networks </a> this we take a combined experimental and computational strategy. To understand drug response, we measure use different technologies to measure the dynamics of gene and protein expression following different drug perturbations, in various genetic backgrounds. Funded by <a href="https://www.standup2cancer.org/"> SU2C </a>,
we are using MAPK inhibition in melanoma as a model to derive a more general framework to model and predict patient response to drug.  Our data and its analysis revealed novel cross-talk with additional pathways that are key determinants influencing the degree of apoptosis following drug treatment. The power of our approach derives from our perturbation-based design, which unveils pathway behaviors that are not apparent without perturbation. We are currently extending our efforts to understand drug response in two critical directions: (1) The role of epigenetics on drug response.  (2) The role of stroma and immune system on drug response. We continue our efforts to build increasingly sophisticated models of drug response based not only on understanding which components go wrong, but also how these go wrong in each individual patient, towards improving cancer therapeutics. 
</d>
</table> </td>	</table></body></html>