<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd"><html><head>  <title>Dana Pe'er Lab </title>  </head>  <body>   <img src="../pics/complogo.jpg" width="980" height="250" border="0"><table width="700" align="center">	 <tbody>  		<TABLE><TR valign="top">  <TD><TABLE  border="0"  cellpadding="4" cellspacing="4"><TR><TD bgcolor="#6699cc" align="center">  <a href="../index.html"  style="text-decoration:none"><b><font color="white" face="Arial" size="3">Home</b></a></td></tr><TR><TD bgcolor="#6699cc" align="center">  <a href="../pub/index.html"  style="text-decoration:none"><b><font color="white" face="Arial" size="3">Publications</b></a></td></tr><TR><TD bgcolor="#6699cc" align="center">  <a href="index.html" style="text-decoration:none"><b><font color="white" face="Arial" size="3">Research</b></a></td></tr><TR><TD bgcolor="#6699cc" align="center">  <a href="../people.html" style="text-decoration:none">  <b><font color="white" face="Arial" size="3">People</b></a></td></tr><TR><TD bgcolor="#6699cc" align="center">  <a href="../collaborations.html" style="text-decoration:none">  <b><font color="white" face="Arial" size="3">Collaborations</b></a></tr></td><TR><TD bgcolor="#6699cc" align="center">  <a href="../openings.html" style="text-decoration:none">  <b><font color="white" face="Arial" size="3">Openings</b></a></tr></td><TR><TD bgcolor="#6699cc" align="center">  <a href="../contact.html" style="text-decoration:none">  <b><font color="white" face="Arial" size="3">Contact</b></a></tr></td></TABLE><td width=30></td><TD width=800><img src="../pics/complexity.jpg" width="700" height="180" border="0"> <br>Over the past few years, <a href = "http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=11923494&query_hl=10&itool=pubmed_docsum"> "genetic genomics" </a> has provided a new and powerful paradigm for understanding the effect of sequence polymorphisms, by analyzing their genome-wide effect on the expression profiles of potential target genes.  This approach haspromise both for providing basic biological insight on gene regulation and as a starting point for understanding human diseases.  <br>Classical computational methods for analyzing these data have been direct extensions of genetic analysis, viewing each gene expression profile as an isolated, quantitative trait.  In collaboration with <a href = "http://ai.stanford.edu/~koller/">Daphne Koller </a> at Stanford, we developed <A href="../pub/pnas2006.pdf"> "Geronemo" </a>a novel computational method designed specifically for gene expression quantitative traits. Our premise is that the influence of genotype on phenotype is induced by fine-grained perturbations to the complex regulatory network that governs a cell's activity.  We provide a computational method that deciphers both the cell's regulatory network and perturbations to it that result from sequence variability.  Our method, builds on our successful <a href="../pub/natgen2003.pdf">Module Networks </a> procedure and offers several significant  advantages over eQTL mapping.<ul><li>Geronemo can distinguish between associations directly induced by sequence variation and those induced by an indirectly via the abundance of a regulator,leading to a better causal understanding of the observed variation.<li> Geronemo exploits the modularity of biological systems, allowing discovery of complex combinatorial regulation programs that are undetectable when considering each gene in isolation.  </ul>We applied Genonemo to a <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=15659551&query_hl=13&itool=pubmed_docsum">dataset </a> containing expression and genotype data for 116 S. cerevisiae strains, generated by crossing a lab strain (BY) with a wild vineyard strain (RM).  Our method produced a range of interesting biological findings regarding both the yeast regulatory network and an understanding how perturbations to it result in variation observed between the strains.  Two of ourmost interesting findings include:<ul><li>Variation in a small number of chromatin modifying factors plays a key role modulating a large fraction of the variance in gene expression between strains.Our global module based analysis suggests that evolutionary forces use changes in a small set of chromatin modification proteins to drive coordinated global changes in the regulatory network. <A href="../pub/pnas2006.pdf"> (PNAS 2006) </a><li>Geronemo predicted a novel mechanism involving regulation of mRNA degradation, connecting Puf3 to P-bodies, which we subsequently verified experimentally.  This connection was uncovered due to our ability to concurrently infer  both the regulatory network and analyze changes to it between strains.</ul>Expression variation among individuals is a powerful resource that is well-suited both for detecting regulatory interactions and uncovering complex phenotypes.  Unlike other types of data (e.g., gene deletions or environmental stimuli), functional assays from divergent strains represent small, natural perturbations to the system, allowing subtle changes to manifest.  Moreover, each individual represents a large set of such perturbations, providing a rich source of statistical variation that helps clarify the signal.  Interestingly, many perturbations are only revealed in the offspring, with the parents showing no variation.  Such data is rapidly accumulating for a number of model systems, yet much of the variation remains unexplained, even by the best models.  We are working on improving and extending Geronemo, and developing entirely new methods. Our computational efforts are aimed at a number of biological questions and directions.<ul><li> Understanding the flow of genetic information, from genotype to phenotype and fitness.We take a multi-layer approach to understand how genotype manifests in phenotypic diversity, using the regulatory network and changes in gene expression patterns and an intermediate to facilitate our understanding.  <li> De-convolving genetic complexity: Despite clear heritability of many phenotypes and disease, association of multi-loci traits has remained an unresolved challenge.  We are developing new approaches to detect causality in multi-loci situations that are obscured by standard techniques.<li> Scaling to mammalian system and clinically relevant problems.  Such scaling entails considerable computational and statistical challenges: mammalian genomesare 100-fold larger, have a higher degree of combinatorial regulation and have a complexlandscape of variation. Nevertheless, our success in yeast bears much promise for itsextension to mammalian systems.</ul> </td>	</table></body></html>